nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/model/DocumentModelResolver.java

```diff
@@ -168,13 +168,16 @@ public boolean validate(Object value) {
 
     @Override
     public boolean validate(Object value, Object context) {
-        MutableBoolean validated = new MutableBoolean();
-        resolve(value, context, (session, docRef) -> {
-            if (session.exists(docRef)) {
-                validated.setTrue();
-            }
-        });
-        return validated.isTrue();
+        if (validation) {
+            MutableBoolean validated = new MutableBoolean();
```

Comment:
Please don't introduce extra indentation when you can just do early exit:

    if (!validation) {
        return true;
    }
    // original code here

This also makes the diff easier to read.